agent/ui_endpoint.go

```diff
@@ -220,8 +224,27 @@ func summarizeServices(dump structs.ServiceDump) []*ServiceSummary {
 
 	for _, csn := range dump {
 		if csn.GatewayService != nil {
-			sum := getService(csn.GatewayService.Service.ToServiceID())
-			sum.GatewayConfig.ListenerPort = csn.GatewayService.Port
+			gwsvc := csn.GatewayService
+			sum := getService(gwsvc.Service.ToServiceID())
+			ingressDNS := serviceIngressDNSName(
+				gwsvc.Service.Name,
+				cfg.Datacenter,
+				cfg.DNSDomain,
```

Comment:
Should you also consider the `cfg.DNSAltDomain`?

Reply:
Yeah I wasn't sure about what to do with the AltDomain. On one hand, if the goal is to provide them with a valid address to contact the service on, then just using the `cfg.DNSDomain` would work and the alt-domain is unnecessary. On the other hand, if we want to provide all valid addresses, then adding the `cfg.DNSAltDomain` would be valid.

Thoughts on that?

Reply:
I think it could be worth adding to clarify that the altdomain is considered for ingress queries.

Reply:
Added the alt domain address to the output.